js/mage/adminhtml/form.js

```diff
@@ -563,22 +561,22 @@ FormElementDependenceController.prototype = {
         // toggle target row
         if (shouldShowUp) {
             var currentConfig = this._config;
-            $(idTo).up(this._config.levels_up).select('input', 'select', 'td').each(function (item) {
```

Comment:
This breaks BC for custom code that uses `this._config.levels_up`, although I do not know of any custom code that uses it,

Reply:
Perhaps I can check if `levels_up` exist and use it if it's true.

Reply:
Not sure if I must to add the check or not.